homeassistant/components/androidtv/__init__.py

```diff
@@ -144,6 +145,8 @@ async def async_connect_androidtv(
         config[CONF_DEVICE_CLASS],
         timeout,
         signer,
+        DEFAULT_TRANSPORT_TIMEOUT_S,
+        False,
```

Comment:
why not just?
```suggestion
        log_errors=False,
```

Reply:
I would have added the param name for all of them, but I just aligned to the syntax used.

Reply:
personally I'd say just add it to all of them and leave out the default

Reply:
Imho providing param name and avoid to import useless constant is better.
But be careful, in this way we will totally loose information about what is causing connection failure.
My suggestion is to find a way to set this value to `false` only if this is not the first try, so error will be logged at least once.

Reply:
I though the approach would be to add as less changes as possible, but I agree on the approach.

Reply:
So let's leave for a next PR, but don't forget this. Connection error log should be useful in many cases.